xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -130,6 +133,7 @@ extern "C" {
     };
     uint32_t m_flags;
 
+    char m_name[256];                    /*!< @brief (optinal) name of the stream, \0 for default handling */
```

Comment:
do you remember that you have to use programs if you want to present the same streams with different bitrates? hls uses programs too and Kodi now supports program selection via a dialog.

Reply:
I have never understood why ffmpeg (hls implementation) expresses different bitrate streams as programs tbh.
I'm currently happy with the multiple stream solution, but yes, I have this in my mind.
Around the MultiBitrateStreams there can be as well "Programs" (=periods in MPD speech) so it would be wrong wasting programs with the streams now.